File: app/src/main/java/com/example/petrsumobile/university/UniversityFragment.java
package com.example.petrsumobile.university;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import com.example.petrsumobile.R;


/**
 * A simple {@link Fragment} subclass.
 */
public class UniversityFragment extends Fragment {

    public UniversityFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        ((AppCompatActivity) getActivity()).getSupportActionBar().setDisplayHomeAsUpEnabled(false);
        ((AppCompatActivity) getActivity()).getSupportActionBar().setTitle("Навигация");

        return inflater.inflate(R.layout.fragment_university, container, false);
    }
}

File: app/src/main/java/com/example/petrsumobile/university/UniversityFragment.kt
package com.example.petrsumobile.university

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.appcompat.widget.Toolbar
import androidx.fragment.app.Fragment
import androidx.navigation.fragment.findNavController
import androidx.navigation.ui.NavigationUI.setupWithNavController
import com.example.petrsumobile.R
import ovh.plrapps.mapview.MapView

/**
 * A simple [Fragment] subclass.
 */
class UniversityFragment : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment_university, container, false)
        val toolbar = view.findViewById<Toolbar>(R.id.toolbar)
        setupWithNavController(toolbar, findNavController())
        

        return view
    }
}